spec/src/main/asciidoc/architecture.adoc

```diff
@@ -33,6 +33,43 @@ This specification documents the component specification contents of each releas
 The specifications for each release are listed in reverse chronological order.
 
 [[microprofile6.0]]
+=== MicroProfile 6.1-RC1
+
+MicroProfile 6.1 still brings in the Jakarta EE 10 Core Profile.
+
+Based on MicroProfile's time-boxed release process, this is a minor release that does not include backward incompatible changes. Like MicroProfile 6.0, this release requires Jakarta EE Core Profile 10.
+
+If you are still dependent on Jakarta EE 9.1, please consider using the https://github.com/eclipse/microprofile/releases/tag/5.0[5.0 release of MicroProfile].
+If you are still dependent on Jakarta EE 8, please consider using the https://github.com/eclipse/microprofile/releases/tag/4.1[4.1 release of MicroProfile].
+If you are still dependent on Java EE 8, please consider using the https://github.com/eclipse/microprofile/releases/tag/3.3[3.3 release of MicroProfile].
+If you are still dependent on Java EE 7, please consider using the https://github.com/eclipse/microprofile/releases/tag/1.4[1.4 release of MicroProfile].
```

Comment:
We should chat about when to eliminate some of these comments. How far should we go back?

Reply:
I think the list is okay. Maybe we can keep just 4. When we move up to EE 11, we can drop Java EE 7.